utils/gyb_syntax_support/ExprNodes.py

```diff
@@ -570,6 +570,82 @@
              Child('RightParen', kind='RightParenToken'),
          ]),
 
+    # SWIFT_ENABLE_TENSORFLOW
+    # Expression generalizing #gradient and #valueAndGradient.
+    # e.g. #gradient(foo(_:_:), wrt: .0, .1)
+    Node('GradientExpr', kind='Expr',
```

Comment:
Note that this name is inconsistent with the base class name for `GradientExpr` and `ValueAndGradientExpr`: `ReverseAutoDiffExpr`. It needs to be changed to `ReverseAutoDiffExpr` to avoid confusion.